hack/verify-golint.sh

```diff
@@ -46,7 +41,7 @@ else
 fi
 
 if [[ -n "${bad_files}" ]]; then
-  echo "golint detected following problems:"
+  echo "!!! golint detected the following problems:"
```

Comment:
If we're changing these messages, we might as well start to use `os::log::error` instead of `"!!!"``